src/bereikbaarheid/admin.py

```diff
@@ -100,9 +100,9 @@ def queryset(self, request, queryset):
         return queryset
 
```

Comment:
Het combineren van changes en linting in dezelfde commit maakt een PR review bijna onmogelijk...